src/main/java/io/github/syst3ms/skriptparser/expressions/CondExprContains.java

```diff
@@ -0,0 +1,70 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.base.ConditionalExpression;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+
+/**
+ * See if a given list of objects contain a given element.
+ * You can also check if a string contains another string.
+ *
+ * @name Contain
+ * @type CONDITION
+ * @pattern %string% [does(n't| not)] contains %string%
+ * @pattern %objects% [do[es](n't| not)] contain[s] %object%
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class CondExprContains extends ConditionalExpression {
+
+    static {
+        Main.getMainRegistration().addExpression(
+                CondExprContains.class,
+                Boolean.class,
+                true,
+                "%string% [1:does(n't| not)] contain[s] %string%",
+                "%objects% [1:do[es](n't| not)] contain[s] %object%"
+        );
+    }
+
+    private Expression<?> first;
+    private Expression<?> second;
+
+    private boolean onlyString;
+
+    @Override
+    public boolean init(Expression<?>[] expressions, int matchedPattern, ParseContext parseContext) {
+        first = expressions[0];
```

Comment:
Expression objects should be cast in init.

Reply:
I didn't do this because it can be either a string or object. I guess I can cast it to object and later to string (I mean Expression<String) of course).

Reply:
Edit: This is not possible and there is actually no real reason to do this.